src/components/TopBar.js

```diff
@@ -299,6 +299,14 @@ const TopBar = Utils.connectStore(authStore, 'authState')(class TopBar extends C
               ...Utils.newTabLinkProps
             }, ['Release Notes'])
           ]),
+          (isTerra() || isFirecloud()) && h(NavSection, {
```

Comment:
Terra or FireCloud? I thought we were doing just Terra (unless Adrian instructed otherwise).